nx-X11/lib/X11/XlibInt.c

```diff
@@ -2753,29 +2890,128 @@ void _XEatData(
     }
 #undef SCRATCHSIZE
 }
+#endif /* !USE_XCB */
 
-/*
```

Comment:
AFAICT, this introduces a regression. _XEatDataWords symbol must continue to exist in libNX_X11, so libXrender does not fail when using it.